### PR TITLE
Final Improvements

### PR DESCRIPTION
**Pull request by Wikitelmex**
- refactor the files in one module folder
- remake the live demo to can see the new implementations working
- remove hardcoded items
- implement task crud function
- implement completed task clear option
- implement function that update the items indexes
- saves to local storage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 // eslint-disable-next-line
```
Comment (rloterh) [inline src/index.js]: - [x] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

File: src/modules/crudtask.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export function onNonEditable(elementId = 0) {
 
 export function orderArray(elementArray = []) {
   elementArray.sort((a, b) => ((a.index > b.index) ? 1 : -1));
-  for (let index = 0; index < elementArray.length; index++) { // eslint-disable-line
+  for (let index = 0; index < elementArray.length; index += 1) {
     elementArray[index].index = index;
```
Comment (rloterh) [inline src/modules/crudtask.js]: - [x] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal to 5).

**Index mismatch in your application:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Alex, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- Good ReadMe
- Great UI design
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/crudtask.js (hunk 2 above)
Comment (Wikitelmex) [reply src/modules/crudtask.js]: updated index in array of tasks

File: src/index.js (hunk 1 above)
Comment (Wikitelmex) [reply src/index.js]: it was disabled just one line to the lodash library, but anyway, this module is not used and deleted the line and the module

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**